slow_tests/test_benchmark.py

```diff
@@ -313,10 +315,12 @@ def benchmark_hif2a(verbose=False, num_batches=100, steps_per_batch=1000):
             )
 
 
+@pytest.mark.skip("Uses deprecated AvgPartialUPartialParam")
```

Comment:
todo: what to do about tests that use `AvgPartialUPartialParam`?

Reply:
Just disable them completely. We don't need to compute them on the fly anymore.

Reply:
Done in 3a3610e.